apps/api/src/app/workflows-v2/usecases/generate-preview/generate-preview.usecase.ts

```diff
@@ -134,17 +134,19 @@ export class GeneratePreviewUsecase {
       return finalPayload;
     }
 
-    const examplePayloadSchema = createMockPayloadFromSchema(workflow.payloadSchema);
+    const examplePayloadSchema = createMockObjectFromSchema(
+      {
+        type: 'object',
+        properties: { payload: workflow.payloadSchema },
+      },
+      true
+    );
 
     if (!examplePayloadSchema || Object.keys(examplePayloadSchema).length === 0) {
       return finalPayload;
     }
 
-    return _.merge(
-      finalPayload as Record<string, unknown>,
-      { payload: examplePayloadSchema },
-      commandVariablesExample || {}
-    );
+    return _.merge(finalPayload as Record<string, unknown>, examplePayloadSchema, commandVariablesExample || {});
```

Comment:
the change was needed because now createMockObjectFromSchema is a more generic function and not `payload` object based.